src/test/kotlin/uk/gov/dluhc/registercheckerapi/service/RegisterCheckServiceTest.kt

```diff
@@ -284,7 +284,7 @@ internal class RegisterCheckServiceTest {
         @ParameterizedTest
         @EnumSource(
             value = CheckStatus::class,
-            names = ["NO_MATCH", "EXACT_MATCH", "MULTIPLE_MATCH", "TOO_MANY_MATCHES"]
+            names = ["NO_MATCH", "EXACT_MATCH", "MULTIPLE_MATCH", "TOO_MANY_MATCHES", "ARCHIVED"]
         )
         fun `should throw RegisterCheckUnexpectedStatusException when existing register check status is not PENDING`(
```

Comment:
If a result is received for a check in ARCHIVED state, the ERO will receive an error with an unexpected status exception, I think this is okay.